p2p/protocol/protocol.go

```diff
@@ -156,96 +175,3 @@ func handleResponse(quaiResp *pb.QuaiResponseMessage, node QuaiP2PNode) {
 	}
 	dataChan <- recvdType
 }
-
-// Seeks the block in the cache or database and sends it to the peer in a pb.QuaiResponseMessage
-func handleBlockRequest(id uint32, loc common.Location, hash common.Hash, stream network.Stream, node QuaiP2PNode) error {
```

Comment:
oh, I see you moved these functions too. Could you explain the move? It is hard for me to compare precisely to confirm no bug has been introduced.